convert2rhel/toolopts.py

```diff
@@ -97,40 +127,63 @@ def _get_argparser():
             "  convert2rhel [--no-rhsm] [--disablerepo repoid]"
             " [--enablerepo repoid] [--no-rpm-va] [--debug] [--restart] [-y]\n"
             "  convert2rhel [-k activation_key | -c conf_file_path] [-o organization] [--pool pool_id | -a] [--disablerepo repoid] [--enablerepo"
-            " repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [-y]"
+            " repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [-y]\n"
+            r"  convert2rhel {analyze}"
             "\n\n"
             "*WARNING* The tool needs to be run under the root user"
         )
         return argparse.ArgumentParser(
-            prog="convert2rhel",
-            conflict_handler="resolve",
-            usage=usage,
-            add_help=False,
+            conflict_handler="resolve", usage=usage, formatter_class=argparse.ArgumentDefaultsHelpFormatter
         )
 
-    def _register_options(self):
-        """Prescribe what command line options the tool accepts."""
-        self._parser.add_argument(
-            "-h",
-            "--help",
-            action="help",
-            help="Show help message and exit.",
+    def _register_commands(self):
+        """ """
```

Comment:
We need a minimal docstring here, at least

Reply:
Reopening this issue since there is no docstring in this method

Reply:
Must have gotten lost in my last commit, I will add it again